src/PKSim.Infrastructure/Serialization/Json/ColorConverter.cs

```diff
@@ -13,7 +13,10 @@ public override void WriteJson(JsonWriter writer, object value, Newtonsoft.Json.
             return;
 
          var c = (Color) value;
-         var hexString = $"#{c.R:X2}{c.G:X2}{c.B:X2}";
+         var hexString = $"{c.R:X2}{c.G:X2}{c.B:X2}";
+         // Only add hex transparency if required as it does not seem to be widely supported by color readers.
+         var preFix = c.A == 255 ? "#" : $"#{c.A:X2}";
+         hexString = $"{preFix}{hexString}";
```

Comment:
well, no idea what this code is doing. So you save the color info as #AARRGGBB instead of #RRGGBB and AA is the transparency info?

Reply:
Yep. It creates an hex strings (it was a request that colors are hex strings if you remember)

But usually transparency is not taken into consideration. That's why I only add it if it's been set by the user so that the hex code is not always weird...

Reply:
Here also write in the comment;-)

Reply:
ok. I assume that this works as expected (never seen transparency info in the rgb hex strings) :)

Reply:
Me too. I didn't try it but the test looks good